idioms-json-2.0/custom_object.json

```diff
@@ -40,12 +40,6 @@
             "last_observed": "2015-07-31T11:24:39.090Z",
             "modified": "2015-07-31T11:24:39.090Z",
             "number_observed": 1,
-            "objects": {
-                "0": {
-                    "ftp_command": "fpuwe8bmsD56ns.bin",
-                    "type": "x-elevator-fooz"
-                }
-            },
```

Comment:
Looks like we lost the ability to capture custom objects? Was this on purpose or bug in these changes?

Reply:
[9:47 AM] Rich Piazza
    I made the decision that if the policy is "add-to-description" that we aren't using custom objects.  I would have put something in the description - but observed-data doesn't have that property
Edited​[9:48 AM] Rich Piazza
    AIS might use that setting - and they don't allow custom objects or extensions